neo-gui/UI/InvokeContractDialog.cs

```diff
@@ -33,53 +33,38 @@ public InvokeContractDialog(InvocationTransaction tx = null)
             }
         }
 
-        public InvocationTransaction GetTransaction()
+        public InvocationTransaction GetTransaction(Fixed8 fee, UInt160 Change_Address = null)
         {
-            Fixed8 fee = Fixed8.Zero;
-            if (tx.Size > 1024)
+            if (fee != Fixed8.Zero) { }
```

Comment:
empty if?

Reply:
Right, unnecessary condition can be removed